core/src/main/scala/com/evolutiongaming/kafka/flow/journal/JournalDatabase.scala

```diff
@@ -66,4 +66,11 @@ object JournalDatabase {
 
     }
 
+  def empty[F[_], K, R](implicit F: Applicative[F]): JournalDatabase[F, K, R] =
```

Comment:
The following style is used elsewhere in `kafka-flow`. I do not mind another style, but we might want to keep it consistent:
```
def empty[F[_]: Applicative, K, R]: JournalDatabase[F, K, R] = ...
```
And then `().pure[F]` is used instead of `F.unit`.

Reply:
Redone in common for the project style